specs/TransportProtocols.md

```diff
@@ -62,6 +62,19 @@ The `POST [endpoint-base]/negotiate` request is used to establish a connection b
   * The `url` which is the URL the client should connect to.
   * The `accessToken` which is an optional bearer token for accessing the specified url.
 
+
+3. A response that contains an `error` which should abort the connection attempt.
```

Comment:
Change abort to stop.

Reply:
Maybe we need a protocol version here @anurse

Reply:
Yeah, it's not a bad idea. But do we want to do the two-part handshake that ASP.NET SignalR does? Where the client reports it's max supported version?

Reply:
ERRR, I hope we don't need to change it that much.